lua/tardis/interiors/sequences/legacy_sequences.lua

```diff
@@ -0,0 +1,21 @@
+--Legacy Interior - Control Sequences (advanced mode)
+
+local Seq = {
+    ID = "legacy_sequences",
+
+    ["legacy_longflightdemat"] = {
+        Controls = {
+            "legacy_helmic",
```

Comment:
Looking at the code it should be keyboard first before helmic otherwise order is good

Reply:
There was like three of those, programmable, manual, and vortex. The one used here is vortex, which starts with the same control. Manual was the one with the keyboard

Reply:
Oh okay, that's fine then!